compiler/src/dmd/clone.d

```diff
@@ -1261,8 +1261,9 @@ FuncDeclaration buildPostBlit(StructDeclaration sd, Scope* sc)
 
         // if this field's postblit is not `nothrow`, add a `scope(failure)`
         // block to destroy any prior successfully postblitted fields should
-        // this field's postblit fail
-        if (fieldsToDestroy.length > 0 && !(cast(TypeFunction)sdv.postblit.type).isnothrow)
+        // this field's postblit fail.
+        // Don't generate it for betterC code since it cannot throw exceptions.
+        if (fieldsToDestroy.length > 0 && !(cast(TypeFunction)sdv.postblit.type).isnothrow && !global.params.betterC)
```

Comment:
```suggestion
        if (fieldsToDestroy.length > 0 && !(cast(TypeFunction)sdv.postblit.type).isnothrow && global.params.useExceptions)
```

Reply:
I thought about this also, but when I looked at this flag it seems that it is by default set to true and it is set to false only if betterC is enabled, so I think that we should just get rid of it and use betterC everywhere.

I see that it was added: https://github.com/dlang/dmd/pull/7321 . I don't really understand the argument there so I still don't understand  why we need `useExceptions`.

However, in this case, it is clear that betterC code is targeted.

Reply:
cc @ibuclaw @WalterBright . What is the point of `useExceptions`?